modules/deploy/partials/cloud/tiers.adoc

```diff
@@ -74,6 +75,18 @@ Amazon Web Services (AWS)::
 | us-west-2
 |===
 --
+Azure::
++
+--
+|=== 
+| Region 
+
+| centralus
+| eastus 
+| norwayeast
+| uksouth
+|===
+--
 ====
 endif::[]
 
```

Comment:
I'm curious why you used a table in the tabs here? Only one column suggests that each tab could simply be a list, no?